bottlecap/src/bin/bottlecap/main.rs

```diff
@@ -213,16 +213,10 @@ fn load_configs() -> (AwsConfig, Arc<Config>) {
 }
 
 fn enable_logging_subsystem(config: &Arc<Config>) {
-    // Bridge any `log` logs into the tracing subsystem. Note this is a global
-    // registration.
-    tracing_log::LogTracer::builder()
-        .with_max_level(config.log_level.as_level_filter())
-        .init()
-        .expect("failed to set up log bridge");
-
+    let env_filter = format!("h2=off,hyper=off,rustls=off,datadog-trace-mini-agent=off,{:?}", config.log_level);
```

Comment:
Not a big deal for now, but an option to enable more verbose debugging might come handy. Especially for `datadog-trace-mini-agent`

Reply:
Although I'd like it, we'd have to upstream fix their info logging, they're logging whenever they send a request, it's not ideal because it shows at the info level